### DISCUSSION TITLE
Powershell not finding my 3.10.1, very new to python

### DISCUSSION THREAD

**Discussion by hyundaidriver** [DIGITALCRIMINALS/OnlyFans#234] [Q&A]
Currently have 3.9.9 through windows app, I do not know how to upgrade to 3.10.1 since I installed 3.9.9 through windows app. If I install 3.10.1 manually, I can't run the commands from the onlyfans master window powershell.

How do I make it recognize that I do have 3.10.1 installed?

**Comment by Kiri31221** [comment]
"If I install 3.10.1 manually, I can't run the commands from the onlyfans master window powershell."

What happens when you try to run the commands from the onlyfans master window powershell if you install 3.10 manually?
Once you said you're new to python, i assume you will have no use for the older version.. Did you tried do remove 3,9 and then install 3.10?

**Reply by hyundaidriver** [reply]
I think I figured out the python 3.10.1 part, but now I am getting an error when trying to download from onlyfans. "AttributeError: 'NoneType' object has no attribute 'pool'"
